From a0aeaa381fca002945e5c4500e245fd7cf3a68ca Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 28 Feb 2014 09:28:28 +0100
Subject: [PATCH] base/init: set the firmware loading timeout to 600

Kernel defaults to 60, but this might be too low, if the machine is
under heavy load, so we increase this to the udev timeout.
---
 modules.d/99base/init | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 1c7bbc7..25a582b 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -135,6 +135,9 @@ source_all cmdline
 getarg 'rdbreak=pre-udev' && emergency_shell -n pre-udev "Break before pre-udev"
 source_all pre-udev
 
+# set firmware loading timeout to 600
+echo 600 > /sys/class/firmware/timeout
+
 # start up udev and trigger cold plugs
 udevd --daemon --resolve-names=never
 
